src/ui/components/atoms/login-register-buttons/LoginRegisterButtons.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import Button from './Button';
+import './styles.scss';
+
+function LoginRegisterButtons() {
+  const handleLoginClick = () => {
+    window.location.href = '/login';
+  };
+
+  const handleRegisterClick = () => {
```

Comment:
another handler shoudl be called outside of the component if you use this approach closes the possibility of reuse this component.